lb.tf

```diff
@@ -3,7 +3,7 @@ resource "azurerm_public_ip" "vault" {
   location              = var.location
   resource_group_name   = azurerm_resource_group.vault.name
   allocation_method     = "Static"
-  domain_name_label     = var.vaul_public_domain_name
+  domain_name_label     = var.vault_public_domain_name
```

Comment:
Are there any current Vault deployments that would break if this typo is fixed?

Reply:
Nop, you can do it